eox_tenant/tenant_wise/proxies.py

```diff
@@ -116,7 +120,35 @@ def get_value_for_org(cls, org, val_name, default=None):
         Returns a configuration value for a microsite or TenantConfig which has an org_filter that matches
         what is passed in.
         """
+        try:
+            return cls.__get_value_for_org(org, val_name, default)
+        except FieldError:
+            return cls.__deprecated_get_value_for_org(org, val_name, default)
+
+    @classmethod
+    def has_configuration_set(cls):
+        """
+        We always require a configuration to function, so we can skip the query
```

Comment:
historically, this used to mean that we where checking if there was a configuration set for the one particular site.
For performance reasons we removed this as it was making a query to the database.
Removing the query was a performance optimization that was worth the fact that we broke the implementation of the function. We can now do the real implementation without the query hit.

Could you please look at the places where this is currently being called? maybe this is worth keeping and fixing and maybe we can just remove it. Maybe there are no more calls to this in the platform and thus is only internal and we can refactor it.

Reply:
@felipemontoya  I have checked it and we can remove it, we only use it in the methods  get_all_orgs, __get_value_for_org and __deprecated_get_value_for_org, and those will be called if the CurrentSiteMiddleware was replace https://github.com/eduNEXT/eox-tenant/blob/cb28cce01203d4c65e46471863e5a89d31a7cccb/eox_tenant/settings/aws.py#L65  and there we have the same condition